velox/common/memory/HashStringAllocator.h

```diff
@@ -335,6 +335,11 @@ class HashStringAllocator : public StreamArena {
     return cumulativeBytes_;
   }
 
+  // Returns the starting sizes of free lists. Allocating one of these
+  // sizes will always be fast because all elements of the free list
+  // in question will fit.
+  static folly::Range<const int32_t*> freeListSizeClasses();
```

Comment:
It is only used by testing so how about s/freeListSizeClasses/testingFreeListSizeClasses/